figures/admin.py

```diff
@@ -1,41 +1,87 @@
 """Provides access to Figures models in the admin dashboard
+
+Filters are in this module because they are specific to the admin console
 """
 
 from django.contrib import admin
+from django.contrib.admin.filters import (
+    AllValuesFieldListFilter,
+    RelatedOnlyFieldListFilter)
+from django.core.urlresolvers import reverse
+from django.utils.html import format_html
 
 import figures.models
 
 
+class AllValuesDropdownFilter(AllValuesFieldListFilter):
+    """Show droplist when there are more than three filter choices for this field
```

Comment:
```suggestion
    """Show dropdown when there are more than three filter choices for this field
```

First time I hear about `droplist`. If it's a synonym to `dropdown`, then I think we probably should use the more common alternative.